vim-patch:27a4632af675

clason opened the pull request:
runtime(filetype): detect *.ck files as Chuck filetype (vim/vim#13888)

closes vim/vim#13886

https://github.com/vim/vim/commit/27a4632af675345f9d3b4f3d66a63756835df8cc

Co-authored-by: Christian Brabandt <cb@256bit.org>

---

(vim commit lacks test)

clason committed:
vim-patch:27a4632af675

runtime(filetype): detect *.ck files as Chuck filetype (vim/vim#13888)

closes vim/vim#13886

https://github.com/vim/vim/commit/27a4632af675345f9d3b4f3d66a63756835df8cc

Co-authored-by: Christian Brabandt <cb@256bit.org>

seandewar reviewed (approved):
Maybe tests will appear later. :innocent:

clason commented:
Yes, I mentioned the lack upstream.

clason committed:
vim-patch:9.1.0042: Missing test for Chuck Filetype

Problem:  Missing test for Chuck filetype
          after commit 27a4632af6753
          (Christian Clason)
Solution: Add a filetype test

https://github.com/vim/vim/commit/c1884c94c5d7246e53a83b03a35a66ce81b49f83

Co-authored-by: Christian Brabandt <cb@256bit.org>

clason commented:
...and there it is 🤗
